### ISSUE TITLE
User not existing, but is existing

### ISSUE THREAD

**Issue by TheDevilsMine** [Gertje823/Vinted-Scraper#60]
My Vinted Download won't let me download some users. It is saying is it not finding any user, but user is active.
For example: 2024-11-24 14:23:49,682 - INFO - User 36607413 does not exist

https://www.vinted.nl/member/36607413

**Comment by Gertje823** [comment]
Just tried but it seems to download the profile just fine.
```
2024-11-24 14:46:04,430 - INFO - Directory created at downloads/Avatars/
2024-11-24 14:46:06,542 - WARNING - Folder already exists at downloads/Avatars/
2024-11-24 14:46:06,574 - INFO - Avatar saved to downloads/Avatars/67108579.jpeg
2024-11-24 14:46:06,612 - INFO - ID=36607413
2024-11-24 14:46:07,283 - INFO - Fetching page 1/1
Fetching page 1/1
2024-11-24 14:46:07,307 - INFO - Total items: 41
2024-11-24 14:46:07,307 - INFO - Successfully created the directory downloads/36607413/
2024-11-24 14:46:07,345 - INFO - Image saved to downloads/36607413/01_0001c_H1xGf9R2c3QjkH3WxXVnTwkx.jpeg
2024-11-24 14:46:07,408 - INFO - Image saved to downloads/36607413/03_00358_x48AhXRpPEiQYKYitPTzuHqS.jpeg
```
Do you use any arguments when running the script?

**Comment by TheDevilsMine** [comment]
On a Virtual machine it works perfectly. Don't know why it says it.

**Comment by MoosMas** [comment]
This probably happens because the requests get blocked by CloudFlare. Instead of the normal response, the request gets a 403 and is presented with a CloudFlare captcha. I'm also having this problem, and haven't been able to bypass it yet. When running it on a local machine, it works fine. But I usually run it on a VPS, and that's where the issue started. I have also tried changing the request's headers to different browsers and platforms, but so far haven't had any luck. My knowledge of scraping isn't extensive enough to provide a solution, unfortunately.

TL;DR: Requests probably get blocked by CloudFlare captcha, and CloudScraper isn't able to solve them. Implementing this will probably fix the issue. It would be awesome if you could look into this!
